BiometricsFingerprint.cpp

```diff
@@ -232,40 +232,23 @@ Return<RequestStatus> BiometricsFingerprint::remove(uint32_t gid, uint32_t fid)
     }
 }
 
-Return<RequestStatus> BiometricsFingerprint::setActiveGroup(uint32_t gid,
-        const hidl_string& storePath) {
-    if (storePath.size() >= PATH_MAX || storePath.size() <= 0) {
-        ALOGE("Bad path length: %zd", storePath.size());
-        return RequestStatus::SYS_EINVAL;
-    }
-    if (access(storePath.c_str(), W_OK)) {
-        return RequestStatus::SYS_EINVAL;
-    }
+int __setActiveGroup(sony_fingerprint_device_t *sdev, uint32_t gid) {
```

Comment:
this should be a private method in the BiometricsFingerprint class.
This is C++, not C :rage2:

Reply:
When you work too much on the kernel....... :P